pennylane/workflow/execution.py

```diff
@@ -160,6 +160,11 @@ def cost_fn(params, x):
     if not isinstance(device, qml.devices.Device):
         device = qml.devices.LegacyDeviceFacade(device)
 
+    ### Apply the user transforms ####
+    if transform_program and getattr(transform_program, "is_user_transform", False):
+        tapes, _ = transform_program(tapes)

```

Comment:
Why don't we need the postprocessing function?

Reply:
I thought user transforms happen before our execution config resolution etc so this kind of transforms should only be modifying tape info instead of process them? Do we have in mind any other user transforms to put here? I might be bit lost here actually